aws/resource_aws_lb_target_group.go

```diff
@@ -120,6 +120,16 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 				}, false),
 			},
 
+			"load_balancing_algorithm_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "round_robin",
```

Comment:
Having this default currently breaks Network Load Balancer Target Groups:

```
--- FAIL: TestAccAWSLBTargetGroup_Protocol_Tls (17.76s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported

--- FAIL: TestAccAWSLBTargetGroup_TCP_HTTPHealthCheck (10.01s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
--- FAIL: TestAccAWSLBTargetGroup_basicUdp (15.40s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
--- FAIL: TestAccAWSLBTargetGroup_defaults_network (25.71s)
    testing.go:654: Step 3 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
--- FAIL: TestAccAWSLBTargetGroup_networkLB_TargetGroup (10.40s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
--- FAIL: TestAccAWSLBTargetGroup_networkLB_TargetGroupWithProxy (11.55s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
--- FAIL: TestAccAWSLBTargetGroup_stickinessWithTCPDisabled (15.60s)
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Error modifying Target Group Attributes: InvalidConfigurationRequest: The provided target group attribute is not supported
```

Since this attribute is shared between both types of target groups, we will need to mark it as `Computed` so Terraform does not try to apply it in all cases.

```suggestion
				Computed: true,
```

Reply:
@bflad Thanks for suggestion!